exporters/inmemory/src/main/java/io/opentelemetry/exporters/inmemory/InMemoryTracing.java

```diff
@@ -57,7 +58,7 @@
    * @return the {@code TracerSdkProvider} passed during construction.
    * @since 0.1.0
    */
-  abstract TracerSdkProvider getTracerProvider();
+  abstract TracerSdkManagement getTracerProvider();
```

Comment:
Methods names should be changed too, shouldn't they?

Reply:
yep. will fix.

Reply:
fixed.